front-end/Nova/api/helper.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosProxyConfig } from 'axios';
+// import https from 'https';
+// import fs from 'fs';
+
+export const backEndSource: string = 'http://127.0.0.1:8080/';
+
+export const httpsProxy: AxiosProxyConfig = {
```

Comment:
What exactly is httpsProxy

Reply:
HTTPS proxy is a struct containing configuration settings for https

Reply:
HTTPS proxy is a struct containing configuration settings for https